generator/syndesis-template.go

```diff
@@ -141,10 +144,10 @@ var productContext = Context{
 	},
 	Tags: tags{
 		Postgresql:    "9.5",
-		PemToKeystore: "1.0",
-		Keycloak:      "1.0",
-		Syndesis:      "1.0",
-		Atlasmap:      "1.30",
+		PemToKeystore: "1.0.0",
+		Syndesis:      "1.0.0",
```

Comment:
Do you really want to swtich from 1.0 to 1.0.0 ? The reason why 1.0 was chose is, that we do not have to update the resource objects when we release a patch (bug fix) release.

The idea, that 1.0 always points to the head of 1.0.x (kind of a symbolic link). 

For feature release when we go to 1.1 (TP2) we would update the scripts and for now require a redeployment.

Does this make sense ? // @jimmidyson @zregvart

Reply:
Perhaps we could be a bit more explicit, like `1.0.latest` ?

Reply:
I was just using the version numbers from the productized images available from internal registry. If that is not correct, then please let me know what they are meant to be and what they finally be published as.

Reply:
The pipeline builds will release 1.0.0, I asked to also publish a 1.0 but this is not technically possible. So we take the 1.0.0 release, import it and _then tag them_. 

Ah, totally forgot to PR my changes  --> Here they are: https://github.com/syndesisio/syndesis-project/pull/130

Reply:
So the import generates (and moves) the 1.0 tag (or 1.1, depending on the full version of the stream).